src/hooks/useShowTime.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import { isTimeBetween } from "../utils/utils";
+
+const useShowTime = (startTime, endTime) => {
+  const [shouldShow, setShouldShow] = useState(() =>
+    isTimeBetween(startTime, endTime)
+  );
+
+  useEffect(() => {
+    console.log("update!");
+    const interval = setInterval(() => {
+      // every 5 seconds, check whether or not the status has changed.
+      // if so, then set state
+      const newShouldShow = isTimeBetween(startTime, endTime);
+      if (newShouldShow !== shouldShow) {
+        setShouldShow(newShouldShow);
+      }
```

Comment:
just to double check, this doesn't do anything with the API call right? I just want to make sure we aren't continually calling to air table. It doesn't look like it but

Reply:
yep! basically all this hook does is return true or false depending on whether the interval is currently set

the useAirTableAPI hook is completely untouched, as well as the entire `RelevantRightNow` component! this is just for the individual cards

Reply:
oh yes I should clarify it only sets state when the previous result was different – ie, when we go from shouldShow = false => shoudShow = true, or vice versa